crates/libs/wallet/src/lib.rs

```diff
@@ -298,6 +301,7 @@ pub fn load_tokens_from_db(
                     keyset_id,
                     secret,
                     c: unblinded_signature,
+                    dleq: None,
```

Comment:
Well, you need to set it.
We want the node to create wads containing proofs with DELQ set.

Reply:
For this you need to store the DELQ field returned by the node during the minting operation.